rules/manifest/applications-gecko-id.js

```diff
@@ -11,20 +11,22 @@ module.exports = class ApplicationsGeckoId extends Rule {
    * @param  {object} flags   Flags via the CLI parser.
    */
   constructor(...args) {
-    super(...args, __filename);
+    super(...args);
   }
 
-  validate() {
+  validate(severity="warn", ...options) {
```

Comment:
This was an amusing change.
Now, each rule can set a default severity (should it be a "warn"ing or "error").
Not sure if it's worth creating some severity enum class/object for "Severity.OFF" vs "Severity.WARN" vs "Severity.ERROR". I usually struggle w/ "warn" vs "warning", but maybe not enough to pollute with a special solution for my idiocy.